### PR TITLE
Update ce installation and prereq

### PR DESCRIPTION
**Pull request by jimmyliao**
1. Add: Prereq git section
2. Update: CE Multinode, Singlenode, Prereq domain name
3. Update git ignore

Note: will update EE with another PR.

cc @gblpedia and @kentwelcome for reviewing.

### CODE REVIEW

File: docs/getting_started/install_primehub_ce.md [hunk 1]
```diff
@@ -14,105 +14,123 @@ sidebar_label: Install PrimeHub CE
...
 
+## Install PrimeHub required binaries
+
 ```bash
-helm repo update
+./primehub/install/primehub-install required-bin
 ```
 
-## Install
+This will install the required commands onto `~/bin`. You should append the `~/bin` to your `PATH` variables.
```
Comment (gblpedia) [inline docs/getting_started/install_primehub_ce.md:55]: Suggest to have one step of appending `~/bin` to $PATH.

File: docs/getting_started/install_primehub_ce.md [hunk 2]
```diff
@@ -14,105 +14,123 @@ sidebar_label: Install PrimeHub CE
...
-     --install \
-     --create-namespace \
-     --namespace hub  \
-     --timeout 30m  \
-     --values primehub-values.yaml
+Prepare two terminals, one to execute the primehub install script, the other to monitor the install progress by watching the pods status.
+
+### Terminal one
+
+Install by `primehub-install create primehub` and specify the version. ex. `v3.6.2`.
```
Comment (gblpedia) [inline docs/getting_started/install_primehub_ce.md:63]: add one sentence 
"Please check the latest stable version."

File: docs/getting_started/install_primehub_ce.md (hunk 1 above)
Comment (jimmyliao) [reply docs/getting_started/install_primehub_ce.md]: added.

File: docs/getting_started/install_primehub_ce.md (hunk 2 above)
Comment (jimmyliao) [reply docs/getting_started/install_primehub_ce.md]: add

**Pull request merged**